### DISCUSSION TITLE
Feat: Support CLI completions in SSH sessions

### DISCUSSION THREAD

**Discussion by brendanfalk** [aws/codewhisperer-command-line-discussions#11] [Feature request]
We now support Autocomplete over SSH, if you want to learn more see the documentation page here: [docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-autocomplete-ssh.html](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-autocomplete-ssh.html)

**Comment by xjose97x** [comment]
This would be great! I frequently SSH into my raspberry pi server. I'd love to get command line completions.

**Comment by AlexEscalante** [comment]
This is so important. I have a main desktop computer which is my terminal to many servers I need to manage.

**Comment by ohmantics** [comment]
This is 100% required for me to migrate from Fig. I frequently ssh into Linux and FreeBSD instances.

**Reply by ohmantics** [reply]
Subjected to the forced migration and the install doesn't work on some machines. Covered in #247.

Also, need a FreeBSD build as I use that heavily.

**Comment by chengmyy** [comment]
Yes, this is what I want. When will this feature released?

**Reply by grant0417** [reply]
We don't have exact timeline but it is in our 2024 goals: https://fig.io/blog/post/fig-is-sunsetting#whats-the-difference-between-fig-and-codewhisperer-for-the-command-line

**Comment by predict-woo** [comment]
I really really need this feature too. Alot of people actually do more work in remote instances.

**Comment by kyc20010501** [comment]
I really really need this feature too!!!

**Comment by angeltonio** [comment]
Same here, I need this feature too, please

**Comment by grant0417** [comment]
We now support Autocomplete over SSH, if you want to learn more see the documentation page here: https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-autocomplete-ssh.html
